proof/crefine/AARCH64/Retype_C.thy

```diff
@@ -2215,7 +2215,9 @@ proof (intro impI allI)
     using al[simplified bit_simps]
     apply -
     apply (rule is_aligned_c_guard[where n="ptBits pt_t" and m=3])
```

Comment:
out of curiosity, what's this 3?

Reply:
It's an old proof, so not quite sure, but I suspect it's the word size of a `pte`